apps/prairielearn/src/lib/editors.ts

```diff
@@ -1282,34 +1268,13 @@ export class CourseInstanceAddEditor extends Editor {
 
     debug('Write infoCourseInstance.json');
 
-    const publishing = {
-      startDate: this.start_access_date
-        ? formatDate(
-            new Date(this.start_access_date.epochMilliseconds),
-            this.course.display_timezone,
-            {
-              includeTz: false,
-            },
-          )
-        : undefined,
-      endDate: this.end_access_date
-        ? formatDate(
-            new Date(this.end_access_date.epochMilliseconds),
-            this.course.display_timezone,
-            {
-              includeTz: false,
-            },
-          )
-        : undefined,
-    };
-
-    const infoJson: Record<string, any> = {
+    let infoJson: Record<string, any> = {
       uuid: this.uuid,
       longName: this.long_name,
```

Comment:
The below spread+reassignment can be inlined:

```suggestion
      longName: this.long_name,
      ...(this.metadataOverrides ?? {}),
```

(GitHub's UI won't let me make a suggestion to remove the code below)